composer.json

```diff
@@ -91,8 +91,8 @@
         "spryker/agent-auth-rest-api": "^1.0.0",
         "spryker/alternative-products-rest-api": "^1.1.2",
         "spryker/api-key-authorization-connector": "^1.0.0",
-        "spryker/app-catalog-gui": "^1.3.0",
-        "spryker/asset": "^1.5.2",
+        "spryker/app-catalog-gui": "^1.4.0",
```

Comment:
regarding https://release.spryker.com/release/pull-request/10302?type=0&released=1 it should be at least `1.4.1`